service/service.go

```diff
@@ -146,6 +149,37 @@ func New(config Config) (*Service, error) {
 			return nil, microerror.Mask(err)
 		}
 	}
+	var newWatcherFactory informer.WatcherFactory
+	{
+
+		zeroObjectFactory := &informer.ZeroObjectFactoryFuncs{
+			NewObjectFunc: func() runtime.Object {
+				var pod api.Pod
+				return &pod
+			},
+			NewObjectListFunc: func() runtime.Object {
+				var podList api.PodList
+				return &podList
+			},
+		}
+		newWatcherFactory = informer.NewWatcherFactory(newK8sClient.Discovery().RESTClient(), "/api/v1/pods/", zeroObjectFactory)
```

Comment:
`/api/v1/pods` returns a `PodList` of all pods if I run it in the k8s api through my browser.
There are errors when running `endpoint-operator` locally though: 
`"...endpoint-operator/vendor/github.com/giantswarm/operatorkit/informer/informer.go:172: watch.Event{Type:\"\", Object:(*api.Pod)(0xc123456789)}} {invalid event}]"`

Reply:
Trying to find out why this happens.

Reply:
The watch event does not seem to have the proper type. I am wondering why the type of the pod is empty. Is this normal? What happens if you do this for service? Does the documentation or the internet says something about this behaviour? See https://github.com/giantswarm/operatorkit/blob/d306448fd014890db29087494a772af93e429d71/informer/informer.go#L171-L173.

Reply:
I see. Nice you found out. I did also not think about the path we use e.g. in the CRD package. See https://github.com/giantswarm/operatorkit/blob/d306448fd014890db29087494a772af93e429d71/crd/crd.go#L175-L177. This was kind of obvious.

Reply:
Yes, this is where I noticed the mistake and felt a bit dumb after :D 
But atleast It got done 👍